Add missing release notes (#127)
dodona-edu/dodona-edu.github.io: en/news/2020-06-03-3.7-release-notes/README.md

---
title: "Dodona 3.7"
description: "Release notes Dodona 3.7"
permalink: '/news/:year/:month/:day/:slug'
date: "2020-06-03"
---

<NewsHeader :title="$frontmatter.title" :date="$frontmatter.date" lang="en" />

> > Dodona 3.7 introduces a new way the evaluate you student's solutions in a structured way.

Reviewer: ```suggestion
> Dodona 3.7 introduces a new way to evaluate your student's solutions in a structured way.
```

![Select students](./students.png)

### Evaluation overview
After selecting your students, you land on the overview page. Here, you can see your evaluation progress for each of the exercises in the series. In addition, you see the evaluation status for each of your students and exercises. The color bar below each icon indicates if that exercise was solved correctly. By clicking on the icon, you go to the evaluation page for that solution on which you can then give feedback. Students can only see the added feedback after you click the "release feedback" button on the overview page.

![Overview](./overview.png)

### Evaluating a solution
If you evaluate a solution, you get to see the submitted code, the correctness of the solution and the individual tests. You can easily add feedback to a solution by adding a global comment or a comment at a specific line of code. Students can only see this feedback after you clicked the release feedback button on the overview page.

> You can easily navigate to the next student using the sidebar on the right. Je can choose to manually mark a solution as done, or do this automatically by clicking on the next button.

Reviewer 2: ```suggestion
You can easily navigate to the next student using the sidebar on the right. You can choose to manually mark a solution as done, or do this automatically by clicking on the next button.
```
